tests/gk-deploy/test_gk_deploy_basic.sh

```diff
@@ -0,0 +1,157 @@
+#!/bin/bash
+
+SCRIPT_DIR=$(cd $(dirname ${0}); pwd)
+STUBS_DIR="${SCRIPT_DIR}/stubs"
+TESTS_DIR="${SCRIPT_DIR}/.."
+INC_DIR="${TESTS_DIR}/common"
+BASE_DIR="${SCRIPT_DIR}/../.."
+DEPLOY_DIR="${BASE_DIR}/deploy"
+
+GK_DEPLOY="${DEPLOY_DIR}/gk-deploy"
+TOPOLOGY="${DEPLOY_DIR}/topology.json.sample"
+
+PATH="${STUBS_DIR}:$PATH"
```

Comment:
I mean cleaned up? Interestingly my PATH looks clean without stub and I don't see a place where it is cleaned up.